Homework/src/main/java/io/codelex/testtwo/exercise2/Basket.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.testtwo.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+    private List<T> list;
```

Comment:
Šis mainīgais nekur netiek izmantots - metodei addToBasket bija jāpievieno elementi šim sarakstam un metodei removeFromBasket jāņem ārā (piemēram pēdējo elementu)